src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -35,10 +35,9 @@ open class LottoNumber private constructor(private val number: Int = 0) {
         }
 
         @Throws(NumberFormatException::class)
-        fun from(numberString: String): LottoNumber {
-            checkValidation(numberString)
-            return lottoNumbers[numberString.toInt()]
-                ?: throw IllegalArgumentException("$MIN_NUMBER~$MAX_NUMBER 사이의 숫자를 입력해 주세요.")
+        fun from(number: String): LottoNumber {
+            checkValidation(number)
+            return from(number.toInt())
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            return from(
                number.toIntOrNull() ?: throw IllegalArgumentException("$MIN_NUMBER~$MAX_NUMBER 사이의 숫자만 입력해 주세요.")
            )
```

Reply:
https://github.com/next-step/kotlin-lotto/commit/5e45a7e542d813db599d5c5da8349549ab42a8de

수정 완료했습니다.